src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PersonContact.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.personrecord.model
+
+import uk.gov.justice.digital.hmpps.personrecord.model.types.ContactType
+
+class PersonContact(
+  val contactType: ContactType,
+
+  val contactValue: String? = null,
+) {
+
+  fun isContactValueNullOrEmpty(): Boolean {
+    return contactValue.isNullOrEmpty()
+  }
```

Comment:
Not much point in constructing the class at all if this is null or empty - can we move this check up?